docs/lib/analytics/configure.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Configure existing resources
+description: Configure AWS Amplify to use existing AWS resources by pointing to them in your configuration.
+---
+
+<inline-fragment platform="android" src="~/lib/analytics/fragments/configure.md"></inline-fragment>
+<inline-fragment platform="ios" src="~/lib/analytics/fragments/configure.md"></inline-fragment>
```

Comment:
```suggestion
<inline-fragment platform="android" src="~/lib/analytics/fragments/configure.md" />
<inline-fragment platform="ios" src="~/lib/analytics/fragments/configure.md" />
```
Any reason these wouldn't work with the single tag?

Reply:
Hm, not sure, I copied-pasted. I'll try it.